spec/lib/bamboozled/employee_spec.rb

```diff
@@ -112,5 +112,26 @@
       expect(location).to eq "https://api.bamboohr.com/api/gateway.php/alphasights/v1/employees/44259"
     end
   end
-end
 
+  describe "#update" do
+    it "updates an employee in BambooHR" do
+      xml = YAML.load_file("spec/fixtures/update_employee_xml.yml")
+      response = File.new("spec/fixtures/update_employee_response.json")
+      details = JSON.parse(File.read("spec/fixtures/update_employee_details.json"))
```

Comment:
Hound likes to crush your morale sometimes but it gets nicer with time :smile_cat: